_extensions/software-licensing/updater-implementation-instantiate.html

```diff
@@ -0,0 +1,70 @@
+---
+layout: sample
+title: Updater Implementation for WordPress Plugins -- Instantiating the Updater Class
+description: Sample code to illustrate how to instantiate the Software Licensing plugin updater class in your plugin.
+collection: extensions
+category: software-licensing
+---
+
+<?php
+
+/*
+Plugin Name: Sample Plugin
+Plugin URI: http://pippinsplugins.com/
+Description: Illustrates how to include an updater in your plugin for EDD Software Licensing
+Author: Pippin Williamson
+Author URI: http://pippinsplugins.com
+Version: 1.0
+License: GNU General Public License v2.0 or later
+License URI: http://www.gnu.org/licenses/gpl-2.0.html
+*/
+
+/**
+ * For further details please visit https://docs.easydigitaldownloads.com/article/383-automatic-upgrades-for-wordpress-plugins
+ */
+
+// this is the URL our updater / license checker pings. This should be the URL of the site with EDD installed
+define( 'EDD_SAMPLE_STORE_URL', 'http://easydigitaldownloads.com' ); // you should use your own CONSTANT name, and be sure to replace it throughout this file
```

Comment:
```suggestion
define( 'EDD_SAMPLE_STORE_URL', 'https://easydigitaldownloads.com' ); // you should use your own CONSTANT name, and be sure to replace it throughout this file
```

Just to set a good example. :D